src/main/java/com/team4/testingsystem/entities/Timer.java

```diff
@@ -25,9 +31,16 @@
     @JoinColumn(name = "test_id", referencedColumnName = "id")
     private Test test;
 
-    public Timer(Test test) {
+    @Column(name = "status")
+    @Enumerated(EnumType.STRING)
+    private Status status;
+
+    @Column(name = "scheduled_time")
+    private Instant scheduledTime;
+
+    public Timer(Test test, Status status, Instant scheduledTime) {
         this.test = test;
+        this.status = status;
+        this.scheduledTime = scheduledTime;
     }
```

Comment:
Isn't it allargsconstructor?

Reply:
Nope, field `id` isn't in the constructor